docs/src/modules/spring/pages/using-acls.adoc

```diff
@@ -0,0 +1,192 @@
+
+Note that requests that have the internet principal are requests that Kalix has identified as coming through the Kalix ingress, according to a configured route. This is identified by mTLS, however it does not imply that mTLS has been used to connect to the ingress from the client in the internet. These are separate hops. To configure mTLS from internet clients, see https://docs.kalix.io/services/using-client-certificates.html[Client certificates]
+
+== Configuring ACLs
+
+Kalix ACLs consist of two lists of principal matchers. One to allow to invoke a method, and the other to deny to invoke a method. For a request to be allowed, at least one principal associated with a request must be matched by at least one principal matcher in the allow list, and no principals associated with the request may match any principal matchers in the deny list.
+
+Here is an example ACL on a method:
+
+```java
+    @PostMapping
+    @Acl(
+       allow = @Acl.Matcher(service = "*"), 
+       deny =  @Acl.Matcher(service = "my-service"))
+    public Effect<String> createUser(@RequestBody CreateEmployee create) {
+     ...
+    }
+```
+
+The above ACL allows traffic to all services except the service called `my-service`. 
+
+To allow all traffic:
+
+```java
+@Acl(allow = @Acl.Matcher(principal = Acl.Principal.ALL) 
+```
+
+To allow only traffic from the internet:
+
+```java
+@Acl(allow = @Acl.Matcher(principal = Acl.Principal.INTERNET) 
+```
+
+To allow traffic from `service-a` and `service-b`:
+
+```java
+ @Acl(allow = {
+        @Acl.Matcher(service = "service-a"), 
+        @Acl.Matcher(service = "my-service")}) 
+```
+
+To block all traffic, an ACL with no allows can be configured:
+
+```java
+ @Acl(allow = {})
+```
+
+=== Sharing ACLs between methods
+
+The above examples show how to configure an ACL on a method. ACLs can also be shared between all methods on a service by specifying them on the class, at type level:
+
+```java
+@Acl(allow = @Acl.Matcher(principal = Acl.Principal.INTERNET))
+public class MyAction extends Action { ... } 
+```
+
+The service's ACL can be overridden by individual methods, by specifying the ACL on the method. Note that an ACL defined on a method completely overrides an ACL defined on a service. It does not add to it. So for example, in the following service:
+
+```java
+@Acl(allow = @Acl.Matcher(service = "service-a"))
+ public class MyAction extends Action {
+    @PostMapping
+    @Acl(allow = @Acl.Matcher(service = "service-b"))
+    public Effect<String> createUser(@RequestBody CreateEmployee create) {
+      return ...;
+    }
+}
+```
+
+The `MyMethod` rpc method will allow calls by `service-b`, but not by `service-a`.
+
+=== Configuring the default policy
+
+The default policy can be configured by specifying a project level annotation in the `Main`, for example, to set a default policy of allowing all local services:
```

Comment:
what means project and service here? I think this issue has been raised before. The distinction between, 
1. project as deployable, 
2. project as project id in Kalix
3. service as project deployable 
4. service as component

I'm quite sure this list is wrong. Shall we try to standardize it and maybe add it to the glossary?

Reply:
I mean. Not right now, but add an issue maybe?

Reply:
Indeed, here is project as deployable.